Source/Particles/Collision/BinaryCollision/DSMC/SplitAndScatterFunc.H

```diff
@@ -329,31 +300,31 @@ public:
                 soa_products_data[0].m_rdata[PIdx::w][species1_index] = p_pair_reaction_weight[i];
 
                 // create a copy of the first product species at the location of species 2
-                const auto product1_index = ioniz_product1_offset + with_product_p_offsets[i];
-                copy_species1[ioniz_product1_list_index](soa_products_data[ioniz_product1_list_index], soa_2, static_cast<int>(p_pair_indices_2[i]),
+                const auto product1_index = products_np_data[2] + with_product_p_offsets[i];
+                copy_species1[2](soa_products_data[2], soa_2, static_cast<int>(p_pair_indices_2[i]),
```

Comment:
Btw, @roelof-groenewald this is orthogonal to this PR, but should this be `copy_species2` instead of `copy_species1`?
```suggestion
                copy_species2[2](soa_products_data[2], soa_2, static_cast<int>(p_pair_indices_2[i]),
```

Reply:
The reason there are two vectors (`copy_species1` and `copy_species2`) is because in the fusion module products are placed at both the colliding particles' positions. In this module the new particles are placed at the "target species" (species1) positions, so in all cases `copy_species1` is used. Does that make sense? I'd be happy to explain that in more detail if needed.

Reply:
OK, thanks for clarifying.

Reply:
Hm, but actually, shouldn't the target species be `species2`, precisely?

Reply:
Ahh, yes, that is true. This should change then, I agree.